Rakefile

```diff
@@ -24,6 +24,10 @@ if Semian.sysv_semaphores_supported?
     ext.ext_dir = 'ext/semian'
     ext.lib_dir = 'lib/semian'
   end
+  Rake::ExtensionTask.new('semian_cb_data', GEMSPEC) do |ext|
```

Comment:
I think I misunderstood/wasn't clear earlier. I think this code belongs in the semian .so, but not in the same source file.